### ISSUE TITLE
Contrast issues

### ISSUE THREAD

**Issue by jechols** [open-oni/open-oni#303]
The bootstrap default link color is pretty bad for accessibility.  The newspaper titles page is particularly bad for us (WAVE flags hundreds of contrast errors).  I will probably fix this in our theme, but we really want to get things like this figured out in a more general case.

On the other hand, anybody who makes their own theme will need to figure out contrast for themselves anyway....

**Comment by jechols** [comment]
Contrast violations appear to have been addressed a while ago, so I'm closing this
